test/nn/test_convolution.py

```diff
@@ -1,4 +1,5 @@
 # Owner(s): ["module: nn"]
+# ruff: noqa: F841
```

Comment:
This file is pretty important @mikaylagawarecki we should fix this one when we have a minute